### PR TITLE
Reduced price of all lighters to a more reasonable value

### LINKED ISSUE #39945: Normal lighters have too much trade value.
# Describe the bug
Lighters are considerbly more expensive than most other items, a full lighter is worth over $100

# Steps To Reproduce
Trade a lighter.

# Expected behavior
For the unrefillable lighter to be reasonably priced.

# Screenshots
image

# Versions and configuration
- OS: Windows
    - OS Version: 10.0 1903
- Game Version: 0.E-1231-g4968f19 [64-bit]
- Graphics Version: Tiles
- Game Language: System language []
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food]
]

# Additional context

### PR DESCRIPTION
**Pull request by TartarusKelvin**
#### Summary
```SUMMARY: Balance "Reduced trading prices for both refillable and non refillable lighters"```

#### Purpose of change

Fixes #39945 - Lighters are considerbly more expensive than most other items, a full lighter is worth over $100 while similar fire starting tools are priced closer to $50

#### Describe the solution
Updated the price and price_postapoc for both lighters 

#### Describe alternatives you've considered
keep the values the same

### CODE REVIEW

File: data/json/items/tool/fire.json [hunk 1]
```diff
@@ -93,8 +93,8 @@
     "description": "This is a disposable plastic lighter.  A lighter must be carried to use various drugs, like cigarettes, or to light things like Molotov cocktails.  You can also use a lighter to light nearby items on fire.",
     "weight": "14 g",
     "volume": "14 ml",
-    "price": 100,
```
Comment (LaVeyanFiend) [inline data/json/items/tool/fire.json:96]: This is $1.00, not $100(which would be 10000) as you said in the summary.

**Comment by TartarusKelvin** [comment]
Whereabouts is the price per charge i must be blind i cant seem to find it

**Comment by TartarusKelvin** [comment]
By my understanding of the bartering system what i have done is correct:

From Item.cpp more accurately item::price a tool which does not have have ammo which a lighter does not has its value reduced proportional to the number of charges (So if it had a value of 100 but had 5/100 charges it would have a true value of 100 * 5/100) by changing the value to 60 the value of a lighter with 1 charge would actually have a value of 60 * 1/100.

However from npctrade.cpp the value you sell for is changed by 0.05*(Seller Barter Skill - Buyer Barter Skill) which gives $1.20 in the issue.

By my changes the values become:
image

which is the expected values

**Comment by Dacendeth** [comment]
This is still a lot, with 100 charges a player should be able to sell a lighter for about 10 dollars not factoring any barter bonuses, considering the refillable one is about 5 dollars filled with 50 charges.

**Comment by TartarusKelvin** [comment]
New updated price is $0.10 per charge so a full lighter should sell for $1.00 before barter bonuses

**Comment by mlangsdorf** [comment]
$1.2 is probably overvalued but it's not grossly out of whack.

**Pull request merged**